doc/src/site/sphinx/tutorials/kerberos_auth.rst

```diff
@@ -67,24 +68,25 @@ You can pass the required properties directly as Spark properties, such as:
     --conf spark.ext.h2o.login.conf=kerberos.conf \
     --conf spark.ext.h2o.user.name=username
 
-And later, you can create ``H2OContext`` without the configuration object as:
+And later, you can create ``H2OContext`` as:
 
 .. code:: python
 
     from pysparkling import *
-    hc = H2OContext.getOrCreate(spark, auth=("username", "password"))
+    conf = H2OConf(spark).setUserName("username").setPassword("password")
+    hc = H2OContext.getOrCreate(spark)
 
 
 Or, you can also use setters available on ``H2OConf`` as:
 
 .. code:: python
 
     from pysparkling import *
-    conf = H2OConf(spark).set_login_conf("kerberos.conf").set_user_name("username").set_kerberos_login_enabled()
-    hc = H2OContext.getOrCreate(spark, conf, auth=("username", "password"))
+    conf = H2OConf(spark).set_login_conf("kerberos.conf").set_user_name("username").set_kerberos_login_enabled().setUserName("username").setPassword("password")
```

Comment:
Username seems to be specified twice `.set_user_name("username")` and `.setUserName("username")`